### PR TITLE
Adding a fits output to file

### PR DESCRIPTION
**Pull request by cam156**
## Description

Added a property to store the output
added an operation to create the output

Fixes: #552

Why was this necessary?

## Changes

Are there any major changes in this PR that will change the release process? no

## Checklist

- [ ] i18n enabled
- [ ] adequate test coverage
- [ ] accessible interface

### CODE REVIEW

**Comment by cam156** [comment]
comment

File: spec/cho/transaction/operations/file/characterize_spec.rb [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  describe '#call' do
+    context 'with a file' do
+      let(:resource) { Work::File.all.first }
+      let(:work_resource) { build(:work, title: 'with a file', member_of_collection_ids: [collection.id]) }
+      let(:work_change_set) { Work::SubmissionChangeSet.new(work_resource) }
+      let(:temp_file) { Rack::Test::UploadedFile.new(Rails.root.join('spec', 'fixtures', 'hello_world.txt')) }
+      let(:resource_params) { { label: 'abc123', file: temp_file } }
+
+      before do
+        mock_fits_for_travis
```
Comment (awead) [inline spec/cho/transaction/operations/file/characterize_spec.rb]: @cam156 in Scholarsphere, don't we download fits and run it? Do we want to do that in CHO too? We can also put it off for later.

File: app/cho/transaction/operations/file/characterize.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+          Success(change_set)
+        rescue StandardError => e
+          Failure("Error characterizing file: #{e.message}")
+        end
+
+        private
+
+          def characterize(change_set)
+            Valkyrie::Storage::Disk
```
Comment (awead) [inline app/cho/transaction/operations/file/characterize.rb]: @cam156 Does this serve any purpose?

Comment (cam156) [reply app/cho/transaction/operations/file/characterize.rb]: @awead no good catch

File: spec/cho/transaction/operations/file/characterize_spec.rb (hunk 1 above)
Comment (cam156) [reply spec/cho/transaction/operations/file/characterize_spec.rb]: I was not certain if we wanted a git submodule again or something else.  I just mocked it for the moment, but if you want the submodule we can.

Comment (awead) [reply spec/cho/transaction/operations/file/characterize_spec.rb]: I've added a ticket #591 and we can decide later. Otherwise, this all looks good to me!

**Pull request merged**